app/controllers/home_controller.rb

```diff
@@ -1,5 +1,10 @@
-class HomeController < ApplicationController  
+class HomeController < ApplicationController
   def index
-    @card = current_user.cards.unreviewed.first
+    @deck = current_user.decks.current.first
+    if @deck.nil?
+      @card = current_user.cards.unreviewed.first
```

Comment:
you have repeated `.cards.unreviewed.first`, how you can invoke it only once and made code better and DRY?